source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -373,83 +387,90 @@ ::do_prolongate_add (const unsigned int                           to_level,
       const unsigned int n_chunks = cell+vec_size > n_owned_level_cells[to_level-1] ?
                                     n_owned_level_cells[to_level-1] - cell : vec_size;
 
-      // read from source vector
-      for (unsigned int v=0; v<n_chunks; ++v)
+      // FIXME: for starters, try a simple loop over blocks. In reality,
+      // one has to do work on several blocks of vectors at a time.
+      for (unsigned int b = 0; b < internal::get_n_blocks(dst); ++b)
```

Comment:
@kronbichler for now I am trying to make this naïve extension to work, but ideally I presume one needs to read into `evaluation_data` several (or all) blocks of a block vector and then feed it to the matrix-free kernels below. But it's not clear to me whether `evaluation_data` should become a `Table<2,...>` or something more complicated to play nicely with `EvaluatorTensorProduct`. Do you have any ideas?